hermeto/core/models/sbom.py

```diff
@@ -23,6 +23,23 @@
 log = logging.getLogger(__name__)
 
 
+Annotator = Literal["organization", "individual", "component", "service"]
+
+
+class Annotation(pydantic.BaseModel):
+    """
+    A comment, note, explanation, or similar textual content
+    which provides additional context to the object(s) being annotated.
+
+    https://cyclonedx.org/docs/1.6/json/#annotations
+    """
+
+    subjects: list[str]
+    annotator: dict[Annotator, dict[str, str]]
+    timestamp: str
```

Comment:
Just a question: Is there any reason not to use the datetime type internally and have a serializer for the correct format? I see we also use str for SPDX annotationDate.

Reply:
I have linked [this](https://docs.pydantic.dev/2.0/usage/types/datetime/#validation-of-datetime-types) in the other PR, the validator looks very permissive and accepting quite a few options, from the looks of it it will be fine with a string representation.